plugins/guides/utils.js

```diff
@@ -99,7 +99,10 @@ export const CLASS_HIERARCHY_RDF_INSTANCES_DEFAULT_TITLE = 'guide.step_plugin.cl
 export const CLASS_RELATIONSHIPS_DEFAULT_TITLE = 'view.class.relationships.title';
 export const REPOSITORIES_CREATE_DEFAULT_TITLE = 'guide.step-action.create-repository';
 export const CREATE_SIMILARITY_INDEX_DEFAULT = 'guide.step-action.create-similarity-index';
-export const ENABLE_AUTOCOMPLETE_DEFAULT_TITLE = 'guide.step-action.enable-autocomplete';
+export const ENABLE_AUTOCOMPLETE_DEFAULT_TITLE = {
```

Comment:
Why you put these labels here? Put them in the steps, please

Reply:
This is constant that is used from many steps.